### ISSUE TITLE
Customizer doesn't load custom /template/style.css

### ISSUE THREAD

**Issue by SGr33n** [Automattic/amp-wp#4565]
Hi,
Defining a custom style.css, example:

```php
function test_amp_custom_style( $file, $type, $post ) {	
    if ( $type === 'style' ) {
        $file = dirname( __FILE__ ) . '/templates/style.php';
    }
	return $file;
}
add_filter( 'amp_post_template_file', 'test_amp_custom_style', 10, 3 );
```

Works good, but then neither the default or the new custom one is loaded when you're on the WordPress customizer.

**Comment by westonruter** [comment]
I believe this may be a duplicate of #4551 being fixed in #4553, to be released next week in v1.5.3.

Please check if the build here fixes the issue: https://github.com/ampproject/amp-wp/pull/4553#issuecomment-611613601

**Comment by SGr33n** [comment]
I can confirm, everything works :)

Thanks @westonruter !!!
